tests/optimizers/abc_test_optimizer.py

```diff
@@ -81,7 +81,14 @@ def test_ftol_effect(self, options, optimizer):
         """Test if setting the ftol breaks the optimization process"""
         opt = optimizer(10, 2, options=options, ftol=1e-1)
         opt.optimize(sphere, 2000)
-        assert np.array(opt.cost_history).shape != (2000,)
+        assert np.array(optimizer.cost_history).shape != (2000,)
```

Comment:
Just out of curiosity, why did you change this line?🤔 Shouldn't it stay as it is?😄

Reply:
Hm you're right. I probably didn't notice it because this test fixture is skipped :smile: 

I'll update the PR with the original code.

Reply:
@whzup I think it's ok now 👍